### PR TITLE
Fix #3384 : Arrange strings file

### LINKED ISSUE #3384: Arrange string resources based on screens
**Describe the bug**

Currently string resources are not organised correctly and are scattered in `strings.xml` file.

This should be fixed by following these points:
1. Make sure that each string is getting used in one specific view only. For example `Home` is part of multiple views like `Navigation Drawer`, `Home-Toolbar`. So for this we should have separate strings for both.
2. All strings related to one activity/fragment should be kept together by mentioning the activity name on top of the strings in a comment. Example: `<!-- ProfileResetPinActivity -->`
3. All activities/fragment sections should be in correct alphabetical order based on activity/fragment name. (The actual strings name are not required to be in alphabetical order but the comments should be in alphabetical order). Example:
```
<!-- HomeActivity -->
<string name="random_string_related_to_home">ABC</string>
<!-- ProfileResetPinActivity -->
<string name="random_string_related_to_profile_resent_pin">ABC</string>
<string name="random_string_related_to_profile_resent_pin_2">XYZ</string>
```

Arrange strings file as per below criteria:

**Screen/Toolbar Titles**
Used in manifest label, toolbar title and activity label test.
Pattern: `activity_name_activity_title`

Example: `administrator_controls_activty_title`

**App Content Strings**
Pattern: `activity_name_view_component_type`

`administrator_controls_activity_profile_management_label`
`administrator_controls_activity_account_actions_label`
`administrator_controls_activity_logout_item_text`
`administrator_controls_activity_app_version_item`

**Shared Strings**
Pattern: `view_component_description`

reading_text_size_item_text_small
reading_text_size_item_medium

**Test Strings**
Pattern: `test_file_name_view_component_type`

Dialogs
Pattern: `dialog_name_view_component_type`

Grouping Pattern: 

  `<!-- AdministratorControlActivity -->`
….
….
  `<!-- GeneralAvailabilityUpgradeNoticeDialogFragment -->`
…..
   `<!-- ProgressDatabaseFullDialog -->`

### PR DESCRIPTION
**Pull request by MohitGupta121**
## Explanation

Fix #3384 : Arrange strings file as per below criteria:

**Screen/Toolbar Titles**
Used in manifest label, toolbar title and activity label test.
Pattern: `activity_name_activity_title`

Example: `administrator_controls_activty_title`

**App Content Strings**
Pattern: `activity_name_view_component_type`

`administrator_controls_activity_profile_management_label`
`administrator_controls_activity_account_actions_label`
`administrator_controls_activity_logout_item_text`
`administrator_controls_activity_app_version_item`

**Shared Strings**
Pattern: `view_component_description`

reading_text_size_item_text_small
reading_text_size_item_medium

**Test Strings**
Pattern: `test_file_name_view_component_type`

Dialogs
Pattern: `dialog_name_view_component_type`

Grouping Pattern: 

  `<!-- AdministratorControlActivity -->`
….
….
  `<!-- GeneralAvailabilityUpgradeNoticeDialogFragment -->`
…..
   `<!-- ProgressDatabaseFullDialog -->`

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

### CODE REVIEW

**Review by MohitGupta121 — COMMENTED** [review]
@adhiamboperes PTAL, I updated this PR as we discussed along with the review suggestions from last https://github.com/oppia/oppia-android/pull/5236#pullrequestreview-1744422007 review comments. You should contiune the review from those comment.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,167 +1,541 @@
...
+  <string name="administrator_controls_fragment_general_label">General</string>
+  <string name="administrator_controls_fragment_edit_account_text">Edit account</string>
+  <string name="administrator_controls_fragment_profile_management_label">Profile Management</string>
+  <string name="administrator_controls_fragment_edit_profiles_text">Edit profiles</string>
+  <string name="administrator_controls_fragment_download_permissions_label">Download Permissions</string>
+  <string name="administrator_controls_fragment_topic_update_on_wifi_text">Download and update only on Wi-fi</string>
+  <string name="administrator_controls_fragment_topic_update_on_wifi_description">Topics will be downloaded and updated only on Wi-fi. Any downloads or updates of cellular data will be queued.</string>
+  <string name="administrator_controls_fragment_auto_update_topic_text">Automatically update topics</string>
+  <string name="administrator_controls_fragment_auto_update_topic_description">Downloaded topics that have new content available will automatically update.</string>
+  <string name="administrator_controls_fragment_app_information_text">App Information</string>
```
Comment (adhiamboperes) [inline app/src/main/res/values/strings.xml]:
```suggestion
  <string name="administrator_controls_fragment_app_information_label">App Information</string>
```

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,167 +1,541 @@
...
+  <string name="administrator_controls_fragment_app_version_text">App Version</string>
+  <string name="administrator_controls_fragment_account_actions_text">Account Actions</string>
```
Comment (adhiamboperes) [inline app/src/main/res/values/strings.xml]:
```suggestion
  <string name="administrator_controls_fragment_account_actions_label">Account Actions</string>
```

**Review by adhiamboperes — CHANGES_REQUESTED** [review]
Thanks @MohitGupta121, I took a pass, the previous comments have been sufficiently adressed. I just had a couple more.

Please fix the merge conflicts as well.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (MohitGupta121) [reply app/src/main/res/values/strings.xml]: Done.

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (MohitGupta121) [reply app/src/main/res/values/strings.xml]: Done.

File: app/src/main/res/values-ar/strings.xml [hunk 3]
```diff
@@ -8,159 +8,145 @@
...
-  <string name="state_end_exploration_button">العودة إلى الموضوع</string>
-  <string name="previous_responses_header">الردود السابقة (%s)</string>
-  <string name="image_interaction_answer_text">يضغط على %s</string>
-  <string name="state_learn_again_button">تعلم مرة أخرى</string>
-  <string name="see_more">اعرض المزيد</string>
-  <string name="see_less">اعرض أقل</string>
-  <string name="FAQs">الأسئلة المتكررة</string>
-  <string name="featured_questions">الأسئلة المميزة</string>
-  <string name="frequently_asked_questions">الأسئلة المتكررة</string>
+  <string name="stop_exploration_dialog_cancel_button_text">إلغاء</string>
```
Comment (adhiamboperes) [inline app/src/main/res/values-ar/strings.xml:43]: Should we not also organize translations by categories like in the English file?

**Review by adhiamboperes — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-ar/strings.xml (hunk 3 above)
Comment (MohitGupta121) [reply app/src/main/res/values-ar/strings.xml:43]: @adhiamboperes no I think as per Ben we can skip translation files for now?

**Comment by MohitGupta121** [comment]
@adhiamboperes @BenHenning could you please reopen this PR and pass your final review @BenHenning

**Comment by adhiamboperes** [comment]
@MohitGupta121, could you please resolve the conflics here?

@XichengSpencer, PTAL for a review pass.

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -1,167 +1,553 @@
...
+  <string name="walkthrough_activity_welcome_text">"Welcome %s!"</string>
+  <string name="walkthrough_activity_topic_header_text">What do you want to learn?</string>
+  <string name="walkthrough_activity_final_title_text">Great</string>
+  <string name="walkthrough_activity_get_started_text">Let’s get started.</string>
+  <string name="walkthrough_activity_yes_text">Yes</string>
+  <string name="walkthrough_activity_no_text">No…</string>
+  <string name="walkthrough_activity_pick_different_topic_text">Pick a \ndifferent topic.</string>
+  <string name="walkthrough_activity_are_you_interested_text">Are you interested in:\n%s?</string>
+  <string name="walkthrough_activity_back_button_text">Back Button</string>
+  <!-- GeneralStrings -->
```
Comment (XichengSpencer) [inline app/src/main/res/values/strings.xml:550]: String in this section maybe needs a bit of alphabetical arrangement

**Pull request closed without merging**